pom.xml

```diff
@@ -87,13 +87,13 @@
         <dependency>
             <groupId>gov.nasa.pds</groupId>
             <artifactId>registry-common</artifactId>
-            <version>1.2.0-SNAPSHOT</version>
+            <version>1.1.1</version>
```

Comment:
Hi @jordanpadams , I used to have snapshot versions (in head of main branch) use snapshot versions of our internal dependencies so that the integration can be tested.

The issue raised by sonatype is now solved in the snapshot version of pds-registry-common so we can have dependency on 1.2.0-SNAPSHOT.